SDWebImageWebPCoder/Classes/SDImageWebPCoder.h

```diff
@@ -7,7 +7,7 @@
  */
 
```

Comment:
Change this line into `#import <SDWebImage/SDWebImage.h>` to works for Static Library user.